modules/ROOT/pages/monitor-connectors.adoc

```diff
@@ -0,0 +1,50 @@
+= Monitor Connectors
+
+You can view performance metrics for supported connectors in your application such as what operation is being performed, how long the operations are taking, and whether they are succeeding or failing.
+
+[NOTE]
+This feature requires a Titanium subscription and the Unified Agent.
```

Comment:
Re Titanium, but we link out to the subscription page from other pages: https://docs.mulesoft.com/monitoring/logs (see under the title). Do we want uniformity (this way or that) everywhere?

Reply:
Added